numba/analysis.py

```diff
@@ -1,2 +1,12 @@
-from numba.core.analysis import * # noqa: F403, F401
-from numba.core.analysis import _use_defs_result # noqa: F401
+from numba.core.errors import (deprecate_moved_module,
+                               deprecate_moved_module_getattr)
+from numba.core.utils import PYVERSION
+
+if PYVERSION > (3, 6):
```

Comment:
Or just:
```suggestion
import sys

if sys.version_info >= (3, 7):
```

Reply:
`>=` is probably better here even if you keep using `PYVERSION`

Reply:
Agree, fixed. Also, for future, Numba uses `PYVERSION`.

Reply:
Any reason you have `PYVERSION == sys.version_info[:2]` and not just `PYVERSION = sys.version_info`?

Reply:
Historical, don't think there's any particular need.